lisp/ess-r-mode.el

```diff
@@ -796,6 +796,7 @@ top level functions only."
   (add-hook 'completion-at-point-functions #'ess-r-package-completion nil 'local)
   (add-hook 'completion-at-point-functions 'ess-filename-completion nil 'local)
   (add-hook 'xref-backend-functions #'ess-r-xref-backend nil 'local)
+  (add-hook 'project-find-functions #'ess-r-project -90 'local)
```

Comment:
This is probably worth a newfeat item.

How is the `-90` depth determined?

Reply:
It's arbitrary, the doc says the default priority is 90, so I just put -90 to make sure that our function comes first. Someone else can impose higher priority with values < -90.

Reply:
I am also thinking it should be part of global `project-find-functions` but not very sure.